### ISSUE TITLE
Order elements before rendering

### ISSUE THREAD

**Issue by flxbe** [flxbe/villages#2]
Right now, The elements are rendered in the order they where added to the app using `app.state.addChild`. On can however attach a `zIndex` to manually order the children. This should be done for the map, all objects as well as the UI.

```js
const sprite = new PIXI.Sprite();

//...

function render() {
  sprite.zIndex = 5;
}
```

Source: https://github.com/pixijs/pixi.js/issues/300

**Comment by kruesemann** [comment]
Suggestion: Initialize zIndex for every object as j * MAP_WIDTH + i where the objects position is tile (i, j). Then change zIndex appropriately every time the position changes.

**Comment by flxbe** [comment]
Alternatively, we could just use the `relY` value as the `zIndex`, as this also increases as the object moves to the front. Also, this value is already computed for every object.

**Comment by flxbe** [comment]
Closed in 567631bcba51dfd767968c4107f46af4a85811a2
